<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>statementshow</class>
 <widget class="QMainWindow" name="statementshow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>430</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1300</width>
    <height>430</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1300</width>
    <height>430</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>811</width>
      <height>261</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;Arial&quot;;</string>
    </property>
   </widget>
   <widget class="QTableView" name="tableView_2">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>100</y>
      <width>341</width>
      <height>261</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;Arial&quot;;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonAll">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>30</y>
      <width>207</width>
      <height>47</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20pt &quot;Cambria&quot;;
color: rgb(85, 0, 255)</string>
    </property>
    <property name="text">
     <string>查看所有房间</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonOne">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>30</y>
      <width>93</width>
      <height>47</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20pt &quot;Cambria&quot;;
color: rgb(85, 0, 255)</string>
    </property>
    <property name="text">
     <string>查看</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>132</width>
      <height>38</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20pt &quot;Cambria&quot;;
color: rgb(85, 0, 255)</string>
    </property>
    <property name="text">
     <string>房间号：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>30</y>
      <width>307</width>
      <height>44</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20pt &quot;Cambria&quot;;
</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>891</x>
      <y>30</y>
      <width>340</width>
      <height>49</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButtonDay">
       <property name="styleSheet">
        <string notr="true">font: 20pt &quot;Cambria&quot;;
color: rgb(85, 0, 255)</string>
       </property>
       <property name="text">
        <string>日报表</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonWeek">
       <property name="styleSheet">
        <string notr="true">font: 20pt &quot;Cambria&quot;;
color: rgb(85, 0, 255)</string>
       </property>
       <property name="text">
        <string>周报表</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonMonth">
       <property name="styleSheet">
        <string notr="true">font: 20pt &quot;Cambria&quot;;
color: rgb(85, 0, 255)</string>
       </property>
       <property name="text">
        <string>月报表</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1300</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
